orcid/en/SUMMARY.md

```diff
@@ -2,5 +2,5 @@
 
 * [Introduction](./introduction.md)
 * [Installation & Setup](./installation-setup.md)
-* [Using the Plugin](./using-plugin.md)
```

Comment:
Is this page being deleted? If so, the md file should be deleted as well.

Reply:
Yes - I meant to remove that as well. It's removed as a part of this commit: https://github.com/pkp/pkp-docs/commit/1c4b0d52444788a3a693797e0ca06921a282f265